src/microformats/parse.ts

```diff
@@ -36,6 +36,18 @@ const getRoots = (node: ParentNode): BackcompatRoot[] =>
 const getId = (node: ParentNode): string | undefined =>
   isMicroformatV2Root(node) ? getAttributeValue(node, "id") : undefined;
 
+const getLanguage = (node: ParentNode): string | undefined => {
+  const lang = getAttributeValue(node, "lang");
+
+  if (lang) {
+    return lang;
+  } else if (node.parentNode) {
```

Comment:
```suggestion
  } else if (node.parentNode && isParentNode(node.parentNode)) {
```

This will fix your type error